<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mcFileDialog</class>
 <widget class="QWidget" name="mcFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>510</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>方正黑体简体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Open...</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
    background-color: rgb(200, 200, 200);
}

QComboBox {
    border: 1px solid gray;
    border-radius: 6px;
    padding: 1px 4px 1px 4px;
	background-color: rgb(255, 255, 255);
    width: 60px;
    text-align: center;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
 	background: rgb(255, 255, 255);
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 6px; /* same radius as the QComboBox */
    border-bottom-right-radius: 6px;
}

QComboBox::down-arrow:hover {
	image: url(:/texture/resources/texture/down-arrow_hover.svg);
    /*border: 1px solid rgb(227, 227, 227);*/
}
 
QComboBox::down-arrow {    
	image: url(:/texture/resources/texture/down-arrow.svg);
	width: 10px;
}

QScrollBar:vertical{
	background-color: transparent;
    width:4px;
}

QScrollBar:horizontal{
	background-color: transparent;
    height:4px;
}

QScrollBar::handle:horizontal{
	background-color: rgba(90, 90, 90, 100);
    height:4px;
    border-radius: 2px;
    margin: 0 0 0 0px;
}

QScrollBar::up-arrow,QScrollBar::down-arrow{
    border:0px;
    height:0px;
    width:0px;
}

QScrollBar::add-line,QScrollBar::sub-line{
    border:0px;
    height: 0px;
    width: 0px;
}

QScrollBar::handle:vertical{
	background-color: rgba(90, 90, 90, 100);
    width:4px;
    border-radius: 2px;
    margin: 0 0 0 0px;
}

QScrollBar::handle:hover{
	background-color: rgba(90, 90, 90, 170);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
    background-color: rgb(235, 235, 235);
}

QLineEdit{
    border-radius: 10px;
    border: 1px solid rgb(166, 166, 166);
	background-color: rgb(240, 240, 240);
    padding-left: 10px;
    padding-right: 10px;
}

QPushButton{
    background-color: rgb(250, 250, 250);
    border-radius:9px;
    border: 1px solid rgb(200, 200, 200);
}

QPushButton:checked{
    background-color: #2196f3;
    color: #fff;
}

QPushButton:checked:hover{
    background-color: #1e88e5;
    color: #fff;
}

QPushButton:hover{
    background-color: rgb(245, 245, 245);
}

QPushButton:disabled{
    background-color: rgb(235, 235, 235);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="backwardButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>26</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/navbar/resources/navbar/arrow-left.svg</normaloff>:/navbar/resources/navbar/arrow-left.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="forwardButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>26</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/navbar/resources/navbar/arrow-right.svg</normaloff>:/navbar/resources/navbar/arrow-right.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="returnBackButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>26</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/navbar/resources/navbar/arrow-up.svg</normaloff>:/navbar/resources/navbar/arrow-up.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="addressBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>9</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
	background-color: rgb(255, 255, 255);
    border-top-right-radius: 0px;
    border-bottom-right-radius: 0px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
 	background: rgb(255, 255, 255);
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 0px; /* same radius as the QComboBox */
    border-bottom-right-radius: 0px;
}</string>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="minimumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>方正黑体简体</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 1px solid gray;
    border-radius: 6px;
    border-top-left-radius: 0px;
    border-bottom-left-radius: 0px;
    border-left: 0px;
    qproperty-icon: url(:/navbar/resources/navbar/refresh.svg);
}

QPushButton:hover{
    qproperty-icon: url(:/navbar/resources/navbar/refresh_checked.svg);
}</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/navbar/resources/navbar/refresh.svg</normaloff>:/navbar/resources/navbar/refresh.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="searchBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>方正黑体简体</family>
         </font>
        </property>
        <property name="placeholderText">
         <string>Search...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>8</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>1</number>
     </property>
     <widget class="QTreeView" name="leftFileView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTreeView{
	background-color: rgb(255, 255, 255);
}

QTreeView::item{
	height: 24px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="rightFileView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>9</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView{
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: rgb(245, 245, 245);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>方正黑体简体</family>
           </font>
          </property>
          <property name="text">
           <string>File Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileNmaeEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>9</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>22</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>方正黑体简体</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
    border-radius: 10px;
    border: 1px solid rgb(166, 166, 166);
	background-color: rgb(240, 240, 240);
    padding-left: 10px;
    padding-right: 10px;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="filterBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>176</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="openButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>86</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>方正黑体简体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background: rgb(80, 80, 80);
    border:0px solid transparent;
	color: rgb(255, 255, 255);
	border-radius: 12px;
	padding: 2px 20px 2px 20px;
}

QPushButton:hover{
	border: 2px solid rgb(100, 100, 100);
    background: #fff;
    color: rgb(80,80,80);
}

QPushButton:disabled{
	background: rgb(150, 150, 150);
}</string>
          </property>
          <property name="text">
           <string>Open</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>86</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>方正黑体简体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background: rgb(80, 80, 80);
    border:0px solid transparent;
	color: rgb(255, 255, 255);
	border-radius: 12px;
	padding: 2px 20px 2px 20px;
}

QPushButton:hover{
	border: 2px solid rgb(100, 100, 100);
    background: #fff;
    color: rgb(80,80,80);
}

QPushButton:disabled{
	background: rgb(150, 150, 150);
}</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>mcFileDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>841</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
